ffi/gn/Cargo.toml

```diff
@@ -0,0 +1,77 @@
+# This file is part of ICU4X. For terms of use, please see the file
+# called LICENSE at the top level of the ICU4X source tree
+# (online at: https://github.com/unicode-org/icu4x/blob/main/LICENSE ).
+
+[package]
+name = "icu_capi_gn"
+description = "GN interface to ICU4X"
+version = "0.1.0"
+authors = ["The ICU4X Project Developers"]
+edition = "2018"
+resolver = "2"
+repository = "https://github.com/unicode-org/icu4x"
+license-file = "LICENSE"
+categories = ["internationalization"]
+# Keep this in sync with other crates unless there are exceptions
+include = [
+    "src/**/*",
+    "examples/**/*",
+    "benches/**/*",
+    "tests/**/*",
+    "include/**/*",
+    "Cargo.toml",
+    "LICENSE",
+    "README.md"
+]
+
+[package.metadata.docs.rs]
+all-features = true
+
+[dependencies]
+icu_capi = { version = "1.0", path = "icu4x_root/ffi/diplomat", default-features = false }
+icu = { version = "1.0", path = "icu4x_root/components/icu" }
+icu_provider = { version = "1.0", path = "icu4x_root/provider/core" }
```

Comment:
Note: Only dependencies that are explicitly listed here are given public GN build targets.